openslides_backend/action/actions/meeting/create_update_delete.py

```diff
@@ -104,6 +115,169 @@
 ]
 
 
+class MeetingCreate(CreateActionWithDependencies):
+    dependencies = [
+        MotionWorkflowCreateSimpleWorkflowAction,
+        GroupCreate,
+        GroupCreate,
+        GroupCreate,
+        GroupCreate,
+        GroupCreate,
+        ProjectorCreateAction,
+    ]
+
+    def base_update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        instance = super().base_update_instance(instance)
+
+        admin_group_fqid_list = [
+            key
+            for key, data in self.datastore.additional_relation_models.items()
+            if key.collection.collection == "group" and data.get("name") == "Admin"
+        ]
+        if len(admin_group_fqid_list) != 1:
+            raise ActionException(
+                "There is no admin group created during meeting.create."
+            )
+        admin_group_fqid = admin_group_fqid_list[0]
+
+        payload = [
+            {
+                "id": self.user_id,
+                "group_$_ids": {str(instance["id"]): [admin_group_fqid.id]},
+            }
+        ]
+        self.execute_other_action(
+            UserUpdate, payload, self.datastore.additional_relation_models
+        )
+        return instance
+
+    def get_dependent_action_payload(
+        self, instance: Dict[str, Any], CreateActionClass: Type[Action], index: int
+    ) -> Dict[str, Any]:
+        if CreateActionClass == MotionWorkflowCreateSimpleWorkflowAction and index == 0:
+            return {
+                "name": "Simple Workflow",
+                "default_workflow_meeting_id": instance["id"],
+                "default_amendment_workflow_meeting_id": instance["id"],
+                "default_statute_amendment_workflow_meeting_id": instance["id"],
+                "meeting_id": instance["id"],
+            }
+        elif CreateActionClass == GroupCreate and index == 1:
```

Comment:
As mentioned in #430: I would prefer if `get_dependent_action_payload` would return an array of action data.

Reply:
Done in  #430

Reply:
Done in  #430